### PR TITLE
Standardized tool setup for Go code

### PR DESCRIPTION
**Pull request by kevgo**
- sets up the standardized [O-Tools](https://github.com/originate/o-tools-node) toolset
- uses gometalinter
- fixes the lint errors

### CODE REVIEW

File: go/exorelay/exorelay_test.go [hunk 1]
```diff
@@ -17,16 +17,18 @@ import (
 func FeatureContext(s *godog.Suite) {
 	var exoInstance *ExoRelay
 	exocom := exocomMock.New()
-	go exocom.Listen(4100)
+	go func() {
+		err := exocom.Listen(4100)
+		if err != nil {
+			fmt.Printf("ERROR: cannot start exocomMock: %s", err)
```
Comment (kevgo) [inline go/exorelay/exorelay_test.go]: This is my quick and dirty attempt at checking the return value of `exocom.Listen`, which the linter correctly calls us out for. Strange that the type checker didn't complain about this.

File: go/exocom-mock/bin/spec [hunk 2]
```diff
@@ -1,7 +1,4 @@
 #!/usr/bin/env bash
 set -e
 
-echo "no tests so far"
-exit
 bin/lint
-go install && go test
```
Comment (trushton) [inline go/exocom-mock/bin/spec]: don't we still want `go test` here?

File: go/exocom-mock/bin/setup [hunk 3]
```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
 set -e
 
-echo "no setup necessary"
+glide install
+go get go get -u gopkg.in/alecthomas/gometalinter.v1
```
Comment (trushton) [inline go/exocom-mock/bin/setup]: remove one of the two `go get`s

File: go/exorelay/exorelay_test.go (hunk 1 above)
Comment (charlierudolph) [inline go/exorelay/exorelay_test.go]: Thoughts on `log.Fatal` here? I don't think we want to continue running if this fails

**Pull request closed without merging**